src/routes/api/api-router.ts

```diff
@@ -130,7 +130,7 @@ const cryptorDebugEndpoint = async (_req: Request, resp: Response) => {
 	try {
 		let data = "";
 		for (let i = 0; i < 10; i++) {
-			data = data + "-" + Math.floor((Math.random() * 10));
+			data = `${data}-${Math.floor((Math.random() * 10))}`;
```

Comment:
nit: pretty sure we can just kill the endpoint

Reply:
This PR is already big.  I will leave it to you to decide to do that clean up in a separate PR if you are confident it can die